shared/src/jvmMain/kotlin/com/mbta/tid/mbta_app/ProjectUtils.kt

```diff
@@ -0,0 +1,269 @@
+package com.mbta.tid.mbta_app
+
+import com.mbta.tid.mbta_app.dependencyInjection.appModule
+import com.mbta.tid.mbta_app.model.ObjectCollectionBuilder
+import com.mbta.tid.mbta_app.repositories.GlobalRepository
+import com.mbta.tid.mbta_app.utils.SystemPaths
+import com.squareup.kotlinpoet.CodeBlock
+import com.squareup.kotlinpoet.FileSpec
+import com.squareup.kotlinpoet.PropertySpec
+import kotlin.io.path.Path
+import kotlinx.coroutines.runBlocking
+import okio.Path
+import okio.Path.Companion.toOkioPath
+import org.koin.core.context.startKoin
+import org.koin.dsl.module
+
+val platformModule = module {
+    single<SystemPaths> {
+        object : SystemPaths {
+            override val data: Path
+                get() = Path("build/projectUtils/data").toOkioPath()
+
+            override val cache: Path
+                get() = Path("build/projectUtils/cache").toOkioPath()
+        }
+    }
+}
+
+/**
+ * Utilities that we may need to run from time to time that need access to the real business logic.
+ * Run with `gradlew jvmRun`.
+ */
+@Suppress("unused")
+object ProjectUtils {
+    object TestDataFilters {
+        val lines = setOf("line-Green", "line-SLWaterfront")
+        val routes =
+            setOf(
+                "15",
+                "67",
+                "87",
+                "CR-Fitchburg",
+                "CR-Haverhill",
+                "CR-Lowell",
+                "CR-Newburyport",
+                "CR-Providence",
+                "Orange",
+                "Red"
+            )
+        val allStopsOnRoutes = setOf("Green-B", "Green-C", "Green-D", "Green-E", "Red")
```

Comment:
question(non-blocking): Why not for all the included routes?

Reply:
There are a lot of bus stops in particular that we don’t need data for, and including all stops from all included routes doubles the size of TestData.kt.